src/types/spot.ts

```diff
@@ -1929,3 +1924,3114 @@ export interface BSwapLiquidity {
+  pendingStrategyType?: number;
+}
+
+export interface NewOrderListOTOResponse {
+  orderListId: number;
+  contingencyType: string;
+  listStatusType: string;
+  listOrderStatus: string;
+  listClientOrderId: string;
+  transactionTime: number;
+  symbol: string;
+  orders: {
+    symbol: string;
+    orderId: number;
+    clientOrderId: string;
+  }[];
+  orderReports: {
+    symbol: string;
+    orderId: number;
+    orderListId: number;
+    clientOrderId: string;
+    transactTime: number;
+    price: string;
+    origQty: string;
+    executedQty: string;
+    cummulativeQuoteQty: string;
+    status: string;
+    timeInForce: string;
+    type: string;
+    side: string;
+    workingTime: number;
+    selfTradePreventionMode: string;
+  }[];
+}
+
+export interface NewOrderListOTOCOParams {
+  symbol: string;
+  listClientOrderId?: string;
+  newOrderRespType?: 'ACK' | 'FULL' | 'RESPONSE';
+  selfTradePreventionMode?: string;
+  workingType: 'LIMIT' | 'LIMIT_MAKER';
+  workingSide: 'BUY' | 'SELL';
+  workingClientOrderId?: string;
+  workingPrice: string;
+  workingQuantity: string;
+  workingIcebergQty?: string;
+  workingTimeInForce?: 'GTC' | 'IOC' | 'FOK';
+  workingStrategyId?: number;
+  workingStrategyType?: number;
+  pendingSide: 'BUY' | 'SELL';
+  pendingQuantity: string;
+  pendingAboveType: 'LIMIT_MAKER' | 'STOP_LOSS' | 'STOP_LOSS_LIMIT';
+  pendingAboveClientOrderId?: string;
+  pendingAbovePrice?: string;
+  pendingAboveStopPrice?: string;
+  pendingAboveTrailingDelta?: string;
+  pendingAboveIcebergQty?: string;
+  pendingAboveTimeInForce?: 'GTC' | 'FOK' | 'IOC';
+  pendingAboveStrategyId?: number;
+  pendingAboveStrategyType?: number;
+  pendingBelowType: 'LIMIT_MAKER' | 'STOP_LOSS' | 'STOP_LOSS_LIMIT';
+  pendingBelowClientOrderId?: string;
+  pendingBelowPrice?: string;
+  pendingBelowStopPrice?: string;
+  pendingBelowTrailingDelta?: string;
+  pendingBelowIcebergQty?: string;
+  pendingBelowTimeInForce?: 'GTC' | 'FOK' | 'IOC';
+  pendingBelowStrategyId?: number;
+  pendingBelowStrategyType?: number;
+}
+
+export interface NewOrderListOTOCOResponse {
+  orderListId: number;
+  contingencyType: string;
+  listStatusType: string;
+  listOrderStatus: string;
+  listClientOrderId: string;
+  transactionTime: number;
+  symbol: string;
+  orders: {
+    symbol: string;
+    orderId: number;
+    clientOrderId: string;
+  }[];
+  orderReports: {
+    symbol: string;
+    orderId: number;
+    orderListId: number;
+    clientOrderId: string;
+    transactTime: number;
+    price: string;
+    origQty: string;
+    executedQty: string;
+    cummulativeQuoteQty: string;
+    status: string;
+    timeInForce: string;
+    type: string;
+    side: string;
+    stopPrice?: string;
+    workingTime: number;
+    selfTradePreventionMode: string;
+  }[];
+}
+
+export interface OrderCountUsageResponse {
+  rateLimitType: string;
+  interval: string;
+  intervalNum: number;
+  limit: number;
+  count: number;
+}
+[];
```

Comment:
This looks a bit weird with an interface also being a top level array. I think this should be a simpler non-array interface:
```
export interface OrderRateLimitUsage {
  rateLimitType: string;
  interval: string;
  intervalNum: number;
  limit: number;
  count: number;
};
```

and then the fn using this type, should say it's using an array of it:
```
  getOrderRateLimit(): Promise<OrderRateLimitUsage[]> {
```

That way it's clear in the implementation that it's an array of this interface. Otherwise it's too hidden that this is actually an array, from a readability perspective.